exercises/resistor-color-duo/description.md

```diff
@@ -4,7 +4,11 @@ If you want to build something using a Raspberry Pi, you'll probably use _resist
 * Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
 To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band has a position and a numeric value. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
 
-In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take two colors as input, and output the correct number.
+In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take color names as input, and output a two digit number. Even when the input is more than two colors!
```

Comment:
```suggestion
In this exercise you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take color names as input and output a two digit number, even when the input is more than two colors!
```

Reply:
Thanks 👍

Reply:
Erik, I copied those into the 1569 PR